todo/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="kr">
```

Comment:
나도 아무 생각없이 익숙한 kr로 했다가 리뷰 받았는데 kr은 Kanuri라는 언어래 ㅋㅋ 
수정합쉬당

```suggestion
<html lang="ko">
```